htdocs/delivery/class/delivery.class.php

```diff
@@ -1044,7 +1044,7 @@ public function setDeliveryDate($user, $delivery_date)
 	 *  @param     int			$hideref        Hide ref
 	 *  @return    int             				0 if KO, 1 if OK
 	 */
-	public function generateDocument($modele, $outputlangs = '', $hidedetails = 0, $hidedesc = 0, $hideref = 0)
+	public function generateDocument($modele, $outputlangs, $hidedetails = 0, $hidedesc = 0, $hideref = 0)
```

Comment:
If you remove the default value, this will break compatibility (fatal error on code that call the method without the outputlang defined).
What about using `  $outputlangs = null`  instead ?

Reply:
impossible to break compatibilty because if somebody use  default parameter that will bug due to 3 lines after

`
		$outputlangs->load("something");
`

And all others function generateDocument in all dolibarr sources are without default value :)

Reply:
If an external module do
generateDocument($modele);
I've got a fatal error.